src/main/java/com/derongan/minecraft/mineinabyss/commands/GUICommandExecutor.kt

```diff
@@ -42,6 +45,16 @@ object GUICommandExecutor : IdofrontCommandExecutor() {
                 GondolaGUI(player).show(player)
             }
         }
+        "storage" {
+            playerAction {
+                MIAConfig.data.hubSection?.let {
+                    if (it.key == WorldManager.getSectionFor(player.location)?.key)
```

Comment:
Any reason you are comparing the keys here and not just the sections?

Reply:
I'm not sure I got it right, but as I see the section comparison will compare every property of it. So it's faster to compare only the keys.

Reply:
This is a very premature optimization, and is bad practice (leads to lower quality of code for unnecessary optimization). If a class defines equals, use it! Also comparisons like this are super cheap in general anyway...

Finally, if there was an issue with how equals was being handled, it would be better to fix the Section class instead. That way anywhere we need to do a similar check we get the benefit.